### PR TITLE
Fixes up issues with tests

### PR DESCRIPTION
**Pull request by cpuguy83**
Unfortunately this requires a dagger fix (on the main branch) to actually work otherwise the tests just hang.... but they work!

### CODE REVIEW

File: package_test.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ var setupSSH string
 //go:embed tests/docker-entrypoint.sh
 var entrypointCmd string
 
-const entrypointVersion = "5ebaa181f866e32e59e37face813cf25b74e8911"
+const entrypointVersion = "892ed9a42ceb5f9a9c7198adfc316da64a573274"
```
Comment (pmengelbert) [inline package_test.go:35]: Was there a relevant change in the entrypoint? Or is this just to keep it at the latest version?

File: package_test.go [hunk 2]
```diff
@@ -47,32 +47,32 @@ func testPackage(ctx context.Context, t *testing.T, client *dagger.Client, spec
...
-	buildOutput, err := do(ctx, client, spec)
+	buildOutput, err := do(ctx, client.Pipeline("Build "+spec.Pkg+" for testing"), spec)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	batsCore, batsHelpers := makeBats(client)
 
-	qemu := testutil.NewQemuImg(ctx, client)
+	qemu := testutil.NewQemuImg(ctx, client.Pipeline("Qemu"))
```
Comment (pmengelbert) [inline package_test.go:57]: Are these `Pipeline` arguments just for logging?

**Review by pmengelbert — APPROVED** [review]
Few questions, otherwise LGTM

File: package_test.go (hunk 2 above)
Comment (cpuguy83) [reply package_test.go:57]: Yes, and with the expirmental dagger TUI it helps *a lot*.

File: package_test.go (hunk 1 above)
Comment (cpuguy83) [reply package_test.go:35]: The change removes the implicit SSH socket forward for docker.sock which didn't work for how we are using it and caused errors.
https://github.com/cpuguy83/qemu-micro-env/compare/5ebaa181f866e32e59e37face813cf25b74e8911...892ed9a42ceb5f9a9c7198adfc316da64a573274

**Pull request merged**